src/components/inputs/toggle-switch.css

```diff
@@ -0,0 +1,52 @@
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 50px;
+  height: 23px;
+}
+
+.switch input {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+.toggle-switch {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #ccc;
+  -webkit-transition: .4s;
+  transition: .4s;
+  border-radius: 2rem;
+}
+
+.toggle-switch:before {
+  position: absolute;
+  content: "";
+  height: 15px;
+  width: 15px;
+  left: 4px;
+  bottom: 4px;
+  background-color: white;
+  -webkit-transition: .4s;
+  transition: .4s;
+  border-radius: 50%;
+}
+
+input:checked + .toggle-switch {
```

Comment:
@Eduardclt01 it's probably best to ensure you're not reaching outside of your component here.
i.e. in this case your CSS will apply to all checked inputs that have a `.toggle-switch` sibling, across the entire app. Which is fine in this project, but, as it scales, you're going to get subtle bugs. Rather do something like this:

`.switch input:checked + .toggle-switch { ... }`

or with the new class naming from the previous comment:

`.toggle-switch input:checked + .toggle-switch__thumb { ... }`